### ISSUE TITLE
You appear to have cloned an empty repository when running "gcloud source repos clone gcp-policies --project="prj-b-cicd-f977""

### ISSUE THREAD

**Issue by leoncozzi** [terraform-google-modules/terraform-example-foundation#615]
### TL;DR

After following 0-bootstrap and the seed and CI/CD projects being deployed correctly, along with the cloud repo folders, I get the following error. I checked and the repos are empty.

warning: You appear to have cloned an empty repository.

### Expected behavior

I should be able to find the following with the repo. cp -RT ../terraform-example-foundation/policy-library/ .

### Observed behavior

After following 0-bootstrap and the seed and CI/CD projects being deployed correctly, along with the cloud repo folders, I get the following error. I checked and the repos are empty.

admin_@cloudshell:~/CFT/terraform-example-foundation/gcp-policies$ gcloud source repos clone gcp-policies --project="prj-b-cicd-f977"
Cloning into '/home/admin_/CFT/terraform-example-foundation/gcp-policies/gcp-policies'...
warning: You appear to have cloned an empty repository.
Project [prj-b-cicd-f977] repository [gcp-policies] was cloned to [/home/admin_/CFT/terraform-example-foundation/gcp-policies/gcp-policies].

As you can see the repos are created but they are empty

admin_@cloudshell:~/CFT/terraform-example-foundation$ gcloud source repos list --project=prj-b-cicd-f977
REPO_NAME: gcp-environments
PROJECT_ID: prj-b-cicd-f977
URL: https://source.developers.google.com/p/prj-b-cicd-f977/r/gcp-environments

REPO_NAME: gcp-networks
PROJECT_ID: prj-b-cicd-f977
URL: https://source.developers.google.com/p/prj-b-cicd-f977/r/gcp-networks

REPO_NAME: gcp-org
PROJECT_ID: prj-b-cicd-f977
URL: https://source.developers.google.com/p/prj-b-cicd-f977/r/gcp-org

REPO_NAME: gcp-policies
PROJECT_ID: prj-b-cicd-f977
URL: https://source.developers.google.com/p/prj-b-cicd-f977/r/gcp-policies

REPO_NAME: gcp-projects
PROJECT_ID: prj-b-cicd-f977
URL: https://source.developers.google.com/p/prj-b-cicd-f977/r/gcp-projects
admin_@cloudshell:~/CFT/terraform-example-foundation$ 

### Terraform Configuration

```hcl
Only variables have been updated during 0-bootstrap stage, in the 0-bootstrap folder. As per the instructions. Nothing else.
```

### Terraform Version

```sh
admin_@cloudshell:~/CFT/terraform-example-foundation$ terraform version
Terraform v1.0.11
on linux_amd64

Your version of Terraform is out of date! The latest version
is 1.1.2. You can update by downloading from https://www.terraform.io/downloads.html
admin_@cloudshell:~/CFT/terraform-example-foundation$
```

### Additional information

No

**Comment by argst274** [comment]
Hi, 

I believe what you're experiencing is by design. In the proceeding sections (1.org 2.env etc) steps include copying the contents from this repo into each of the listed repos, instructing you to modify the variables and push them back up and triggering the cloud build process.

hope that helps.

Alan

**Comment by bharathkkb** [comment]
Hi @leoncozzi @argst274 is correct. The expectation is that you would clone them and then copy contents of each stage. Feel free to reopen if you have any questions.
